modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/RaftGroupServiceImpl.java

```diff
@@ -645,7 +656,7 @@ else if (th instanceof SMFullThrowable)
      * @param t The throwable.
      * @return {@code True} if this is a recoverable exception.
      */
-    private boolean recoverable(Throwable t) {
+    public static boolean recoverable(Throwable t) {
```

Comment:
According to our new approach of splitting sendWithRetry inner retries and outer rebalance related it seems incorrect to expose raft inner retry criterion.

Reply:
I've extracted the same method for the TableManager.